notification-service/src/main/java/org/java/mentorship/notification/controller/NotificationController.java

```diff
@@ -0,0 +1,39 @@
+package org.java.mentorship.notification.controller;
+
+import lombok.Generated;
+import lombok.RequiredArgsConstructor;
+import org.java.mentorship.contracts.notification.dto.Notification;
+import org.java.mentorship.notification.service.NotificationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+@Controller
+@RequiredArgsConstructor
+public class NotificationController {
+
+    private final NotificationService notificationService;
+
+    @GetMapping("/notifications/{id}/web")
```

Comment:
Not quite restful
If you need the web notification, one would do /notifications?type=WEB&userId={id}